src/Components/Pages/FormikContextElements/Fees.tsx

```diff
@@ -35,8 +35,11 @@ const FeesFormikWrapped: React.FC<IFeesFormikWrapped> = ({
           <Typography component="p">Transfer Amount:</Typography>
           <Typography component="p">
             {Number(
-              (values as Record<string, string | number>)[amountFormikName],
-            )?.toFixed(3)}{' '}
+              (values as Record<string, string>)[amountFormikName].replaceAll(
+                ',',
```

Comment:
do we already take care of spaces and other chars as well ?

Reply:
Yup. If there are any spaces or any non-decimal characters, the form will show `Input invalid` when the user tries to submit.